common/v2/features/SendAssets/stateFactory.tsx

```diff
@@ -125,7 +126,14 @@ const TxConfigFactory: TUseStateReducerFactory<State> = ({ state, setState }) =>
       .finally(cb);
   };
 
-  const handleSignedTx: TStepAction = (payload: ISignedTx, cb) => {
+  const handleSignedTx: TStepAction = (payload: any, cb) => {
+    let signedTx = payload;
+    // Used when signedTx is a buffer instead of a string.
+    // Hardware wallets return a buffer.
+    if (typeof signedTx === 'object') {
+      signedTx = bufferToHex(signedTx);
```

Comment:
1. Can we find the `ethers.js` alternative? It would be nice to remove the duplicate lib.
2. Can we move the conversion into the wallet logic (`signRawTransaction` instead of the SendAssetsFlow) so the interface can . always manipulate the same type?

Reply:
1) Done.
2) It's actually always a buffer, so i swapped to using ethers.js's `Arrayish` type.